cvat-core/src/server-response-types.ts

```diff
@@ -71,7 +72,9 @@ export interface SerializedProject {
     status: ProjectStatus;
 }
 
-export type TasksFilter = ProjectsFilter & { ordering?: string; }; // TODO: Need to clarify how "ordering" is used
+export type TasksFilter = ProjectsFilter & {
```

Comment:
Why did you remove this comment? It is still not clear..

Reply:
Do you mean, it's not clear in the UI? From the server side it's quite simple and straightforward.